src/test/java/core/basesyntax/service/impl/ReportCreatorImplTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportCreator;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportCreatorImplTest {
+    private static final String SEPARATOR = System.lineSeparator();
```

Comment:
This constant is redundant. We already know that this is a separator because of its name)